.gitattributes

```diff
@@ -1 +1 @@
-* text=auto
+* text=auto eol=lf
```

Comment:
This apparently was fixed:

- <https://github.com/git/git/blob/master/Documentation/RelNotes/2.10.0.txt#L248>
- <https://stackoverflow.com/questions/29435156/what-will-text-auto-eol-lf-in-gitattributes-do>

Reply:
> The
> combination has been fixed to be equivalent to doing
> $ git config core.autocrlf true

Yeah that's what we want (except with `input` instead). The release notes are saying that `text=auto eol=lf` previously behaved like `text eol=lf` which marks every file as text instead of detecting text or binary.

Reply:
This commit was also supposed to be to master because I am the dumb